### ISSUE TITLE
attribute not supported in HTML5: data-lang

### ISSUE THREAD

**Issue by tbee** [UniKnow/AsciiDoclet#2]
First of all, thank you and @chrisvest for trying to fix this asciidoc vs Java 11 debacle. I've decided to post an issue here because yours seems most current (with asciidoc updated). 

I'm getting the error shown in the subject when generating the Javadoc. It occurs on the first line of the asciidoced Javadoc comment, without any noticeable reference to what it actually says. So it seems to be related to what is generated by asciidoctor. Two small modules did compile, before one stumbled over this, which confuses me.

Have you encountered this before?

**Comment by tbee** [comment]
After a bit of research it seems the error has to do with embedded code samples.

**Comment by msche** [comment]
Hi tbee, sorry for my late response but been occupied by my job for a long while... Could you post the embedded code samples that caused the exception

**Comment by tbee** [comment]
Oh my, that is a while back. I think I worked around it by uploading the sources as the documentation to Maven (how nasty). Let me see if I can still reproduce that.

**Comment by msche** [comment]
Sorry for my lack of responding back then.

To quicker indicate what is going wrong during the generation of the javadoc I added a additional log statements showing in the console the source line at which the generation failed. I just released it to the maven repo (5 minutes ago) as version 2.1.1. Hope that makes finding these kind of faults in the javadoc in the future easier.

ps: will take probably a while before the new release is available in maven repo.
